### PR TITLE
Update workflows

### PR DESCRIPTION
**Pull request by pjvv**
Changes proposed in this pull request:

* update workflows

### CODE REVIEW

**Comment by pjvv** [comment]
Can ignore the release version check since there is no code change.

File: .github/workflows/automated_release.yml [hunk 1]
```diff
@@ -33,15 +33,13 @@ jobs:
         run: echo "RELEASE_VERSION=$([ ${GITHUB_REF##*/} = "develop" ] && printf '%s\n' $(cat .release-version)-develop || printf '%s\n' $(cat .release-version))" >> $GITHUB_ENV
 
       - name: Create release
-        id: create_release
-        uses: actions/create-release@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        uses: softprops/action-gh-release@v1
```
Comment (KatyTaylor) [inline .github/workflows/automated_release.yml:36]: What's the difference between these two? (softprops/action-gh-release@v1 vs actions/create-release@v1)

**Review by KatyTaylor — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -35,6 +33,25 @@ jobs:
 
       - name: Lint using ESLint
         run: yarn lint .
+  test:
+    runs-on: ubuntu-18.04
```
Comment (harrietc52) [inline .github/workflows/ci.yml:37]: Does it want to be testing on the latest ubuntu version too?

**Review by harrietc52 — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 2 above)
Comment (pjvv) [reply .github/workflows/ci.yml:37]: Yes thanks good spot.

**Pull request merged**